src/rtcSessionHelpers.ts

```diff
@@ -36,6 +36,7 @@ export function makeActiveFocus(): LivekitFocusActive {
   };
 }
 
+// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
I suggest `Promise.resolve(...)`-ing the return value here as well. That way, the thing that makes eslint happy is closer to the comment at the bottom of the function about how we want to await something.